lib/matplotlib/animation.py

```diff
@@ -386,16 +400,21 @@ def isAvailable(cls):
         if not bin_path:
             return False
         try:
-            p = subprocess.Popen(bin_path,
-                             shell=False,
-                             stdout=subprocess.PIPE,
-                             stderr=subprocess.PIPE,
-                             creationflags=subprocess_creation_flags)
-            p.communicate()
-            return True
+            p = subprocess.Popen(
+                bin_path,
+                shell=False,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                creationflags=subprocess_creation_flags)
+            return cls._handle_subprocess(p)
         except OSError:
             return False
 
+    @classmethod
```

Comment:
No particular reason I think I was under the false impression that can't override a staticmethod in a subclass. In any case I think it makes sense to leave it as is just in case anyone needs access to the data in `cls` in the future.